### PR TITLE
Feat/site 417/style improvements

### PR DESCRIPTION
**Pull request by lukefrogger**
## What's here
- Refactored the FTC data file into 3 files
- Added and updated FTC pages
- Added the color palettes and font selection (not wired up)

#### Updates not pushed
- Removed the unused `useGoogleFonts` hook

### CODE REVIEW

File: packages/sitebuilder/src/wizards/first-time-configuration/screens/ContentTone.tsx
Comment (chumpgrub) [inline packages/sitebuilder/src/wizards/first-time-configuration/screens/ContentTone.tsx:1]: There's a bunch of commented-out code in this file. Can the code be removed?

File: packages/sitebuilder/src/wizards/first-time-configuration/data/ftc-steps.tsx
Comment (chumpgrub) [inline packages/sitebuilder/src/wizards/first-time-configuration/data/ftc-steps.tsx:1]: There's some commented-out code that maybe can be removed.

**Review by chumpgrub — APPROVED** [review]
Lots of great work, Luke! 😍  I found some fussy stuff to comment on, but nice job.

File: packages/sitebuilder/src/wizards/first-time-configuration/screens/ContentTone.tsx
Comment (lukefrogger) [reply packages/sitebuilder/src/wizards/first-time-configuration/screens/ContentTone.tsx:1]: The commented code is stuff that's being worked. This push is only to a partial for he whole changes.

**Pull request merged**